snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/validation/detail/SnomedValidationIssueDetailExtension.java

```diff
@@ -273,7 +274,9 @@ private void extendConceptIssueLabels(BranchContext context, Collection<Validati
 				.build()
 				.execute(context);
 			
-			Map<String, SnomedDescription> preferredDescriptions = SnomedDescriptionUtils.indexBestPreferredByConceptId(descriptions, locales);
+			@SuppressWarnings("unchecked")
+			ListMultimap<String, String> languageMap = (ListMultimap<String, String>) context.service(TerminologyResource.class).getSettings().get(SnomedTerminologyComponentConstants.CODESYSTEM_LANGUAGE_CONFIG_KEY);
```

Comment:
`SnomedDescriptionUtils` sounds like a good place for a new utility method that extracts the language setting from a terminology resource when a `context` parameter is given. Please replace all occurrences of this line with such a method, it should be reachable from all places where you had to add them.